viadot/tasks/azure_sql.py

```diff
@@ -245,26 +246,30 @@ class AzureSQLDBQuery(Task):
 
     def __init__(
         self,
+        if_failed: Literal["break", "skip"] = "break",
         credentials_secret: str = None,
         vault_name: str = None,
         *args,
         **kwargs,
     ):
         self.credentials_secret = credentials_secret
         self.vault_name = vault_name
+        self.if_failed = if_failed
 
         super().__init__(name="azure_sql_db_query", *args, **kwargs)
 
     def run(
         self,
         query: str,
+        if_failed: Literal["break", "skip"] = "break",
         credentials_secret: str = None,
         vault_name: str = None,
     ):
         """Run an Azure SQL Database query
 
         Args:
             query (str, required): The query to execute on the database.
+            if_failed (Literal["break", "skip"], optional): What to do if one of the subqueries fails. Defaults to "break".
```

Comment:
Here should be information about the query - how the query must look like (line/command ending with `;`)